util/occamygen.py

```diff
@@ -391,11 +401,13 @@ def main():
         "narrow_xbar_quadrant_s1": narrow_xbar_quadrant_s1,
         "soc_regbus_periph_xbar": soc_regbus_periph_xbar,
         "apb_hbi_ctl": apb_hbi_ctl,
+        "apb_hbm_cfg": apb_hbm_cfg,
         "cfg": occamy.cfg,
         "cores": nr_s1_quadrants * nr_s1_clusters * nr_cluster_cores + 1,
         "nr_s1_quadrants": nr_s1_quadrants,
         "nr_s1_clusters": nr_s1_clusters,
-        "nr_cluster_cores": nr_cluster_cores
+        "nr_cluster_cores": nr_cluster_cores,
+        "hbm_channel_size": HBM_CHANNEL_SIZE
```

Comment:
I cannot find `hbm_channel_size` being used anywhere — or am I wrong?

Reply:
Yeah indeed, it is needed in the private repo `snitch-pd` :-)

Reply:
👍